evacuees/src/API/EMBC.Registrants.API/RegistrationsModule/RegistrationManager.cs

```diff
@@ -27,7 +28,8 @@ namespace EMBC.Registrants.API.RegistrationsModule
 {
     public interface IRegistrationManager
     {
-        Task<string> RegisterNew(AnonymousRegistration registration);
+        Task<string> CreateRegistrationAnonymous(AnonymousRegistration registration);
```

Comment:
I prefer a polymorphic way of sending messages to managers as it allows to expand future development under the manager rather than proliferate methods per use case.